docs/run-node/run-testnet.md

```diff
@@ -6,7 +6,7 @@ order: 7
 
 The `simd testnet` subcommand makes it easy to initialize and start a simulated test network for testing purposes. {synopsis}
 
-In addition to the commands for [running a node](./run-node.html), the `simd` binary also includes a `testnet` command that allows you to start a simulated test network in-process or to initialize files for a simulated test network that runs in a separate process. 
+In addition to the commands for [running a node](./run-node.html), the `simd` binary also includes a `testnet` command that allows you to start a simulated test network in-process or to initialize files for a simulated test network that runs in a separate process.
```

Comment:
Currently, the important fix (migrateDenom) is hidden inside a lot of noise. I prefer if all the markdown lints could be done in a separate PR.

Reply:
ok, will extract the fix.